ACE/ace/OS_NS_string.cpp

```diff
@@ -101,18 +101,19 @@ ACE_OS::strerror (int errnum)
   errno = 0;
   char *errmsg = 0;
 
-#if defined (ACE_HAS_TR24731_2005_CRT)
+# if defined (ACE_HAS_TR24731_2005_CRT)
   errmsg = ret_errortext;
   ACE_SECURECRTCALL (strerror_s (ret_errortext, sizeof(ret_errortext), errnum),
                      char *, 0, errmsg);
   if (errnum < 0 || errnum >= _sys_nerr)
     g = EINVAL;
 
   return errmsg;
-#elif defined (ACE_WIN32)
+# else
+#  if defined (ACE_WIN32)
```

Comment:
Why is this changed?

Reply:
The previous version results in dead code, which causes a warning from the compiler (based on warning level).